src/main/java/com/google/sps/servlets/ServletUtils.java

```diff
@@ -49,15 +49,17 @@
   public static Optional<String> getRequestParam(
       HttpServletRequest request, HttpServletResponse response, String inputName)
       throws IOException {
+
     String input = request.getParameter(inputName);
```

Comment:
This is not related to this PR, I think I commented before but it got lost in the shuffle. This looks like it's just a collection of static utility functions. Please add a private constructor with a comment
// Private constructor to prevent instantiation. 

See:
http://go/ej3e#page=40